src/axom/sidre/examples/sidre_data_vs_metadata.cpp

```diff
@@ -60,12 +60,17 @@ int main(int argc, char* argv[])
   // group has one view and the datastore has one buffer (holding the
   // view's data).
   //
-  // Initialize the elements of the integer array.  Then, deallocate the view.
+  // Initialize the elements of the integer array.
   //
-  // The view and its description of the data remains, but the buffer
-  // holding the integer array is destroyed on de-alloction since there
+  // Then, deallocate the view. The view and its description of the data
+  // remains, but the buffer holding the array is deallocted since there
```

Comment:
```suggestion
  // remains, but the buffer holding the array is deallocated since there
```

Reply:
Making changes as you suggest them. Will push all in one commit. Keep 'em comin'.